### ISSUE TITLE
foreign key is shown twice when adding a compund unique constraint

### ISSUE THREAD

**Issue by Beiri22** [directus/directus#6661]
### Describe the Bug

Following scenario: an Artwork(title, technique, size, ...) has multiple copies (category, number, price, buyer, ...).
So far, everything works. But, I need to ensure, that the combination of the foreign key artwork and the category + number of the copy is unique. As it is not possible (yet?) to create composite unique constraints from within directus I created the composite unique constraint directly in the database.

But:
1.) Within the datamodel of copy, the foreign key artwork is now shown twice. That is not intended.
2.) The unique-constraint is enforced, but the error message is not very useful. An information liek "the unique constraint on columns x, y and z is violated" would help here.

The doubled foreign key "artwork" is only once the the system table directus_fields.

### To Reproduce

Create the models Artwork(title, technique, size, ...) and copies (category, number, price, buyer, ...) with an 1:n relationship, so that copies references artwork.

Add a composite unique constraint for copies(artwork, category, number) in the database directly.

Open the model view.

### What version of Directus are you using?

rc83

### What version of Node.js are you using?

12

### What database are you using?

i think mysql, via phpmyadmin

### What browser are you using?

Chrome

### What operating system are you using?

Linux

### How are you deploying Directus?

Linevast hosting

**Comment by Beiri22** [comment]
Seems to be the as https://github.com/directus/directus/issues/6204

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/6204
